### PR TITLE
Rename `Replicated` to `replicatedDist`

### PR DESCRIPTION
**Pull request by bradcray**
This renames `Replicated` to `replicatedDist`, similar to what we've done for `Block`, `Cyclic`, and `Stencil` in previous PRs.

### CODE REVIEW

File: modules/dists/ReplicatedDist.chpl [hunk 1]
```diff
@@ -93,13 +93,13 @@ distribution.
 
 **Initializer Arguments**
 
-The ``Replicated`` class initializer is defined as follows:
+The ``replicatedDist`` class initializer is defined as follows:
```
Comment (jabraham17) [inline modules/dists/ReplicatedDist.chpl:96]: I think "class" can be dropped here

**Review by jabraham17 — APPROVED** [review]
Looks good to me!

File: modules/dists/ReplicatedDist.chpl (hunk 1 above)
Comment (bradcray) [reply modules/dists/ReplicatedDist.chpl:96]: D'oh, yes!  Thanks!

**Comment by mppf** [comment]
@bradcray - note that I've just merged #23356 which creates a merge conflict here.

**Pull request merged**